### PR TITLE
feat(question-type-cursor): #FOR-457 Add response view

### PR DESCRIPTION
**Pull request by c4role**
## Describe your changes
Add response view to question cursor type

## Checklist tests
Create a question with a question type cursor, and then check response.

## Issue ticket number and link
[FOR-457] https://jira.support-ent.fr/browse/FOR-457

## Checklist before requesting a review (magic string, indentation, comment/documentation...)

- [x] I have detailed the tests to do in my feature/fix in order to prevent consequents regressions (must specify in **Checklist tests**)
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (API Doc etc...) - (must specify in **Description** for target version)
- [ ] If it is a consequent feature, I have added thorough tests.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been added to this project (must specify in **Description**)

### CODE REVIEW

File: formulaire/src/main/java/fr/openent/formulaire/controllers/ResponseController.java [hunk 1]
```diff
@@ -236,6 +234,9 @@ else if (question.getMatrixId() == null &&
                             try { timeFormatter.parse(response.getString(ANSWER)); }
                             catch (ParseException e) { e.printStackTrace(); }
                         }
+                        if (question.getQuestionType() == 11) {
```
Comment (JinDouang) [inline formulaire/src/main/java/fr/openent/formulaire/controllers/ResponseController.java:237]: Magic number :  

```suggestion
                        if (question.getQuestionType() == QuestionTypes.CURSOR.getCode()) {
```

File: formulaire/src/main/java/fr/openent/formulaire/controllers/ResponseController.java [hunk 2]
```diff
@@ -375,6 +376,10 @@ else if (question.getInteger(MATRIX_ID) == null &&
                             try { timeFormatter.parse(response.getString(ANSWER)); }
                             catch (ParseException e) { e.printStackTrace(); }
                         }
+                        if (question_type == 11) {
```
Comment (JinDouang) [inline formulaire/src/main/java/fr/openent/formulaire/controllers/ResponseController.java:379]: Magic number

```suggestion
                        if (question_type == QuestionTypes.CURSOR.getCode()) {
```

Comment (c4role) [reply formulaire/src/main/java/fr/openent/formulaire/controllers/ResponseController.java:379]: Est-ce qu'au passage je modifie les autres ou c'est mieux de le faire à un autre moment sur un autre commit ?

Comment (qperie) [reply formulaire/src/main/java/fr/openent/formulaire/controllers/ResponseController.java:379]: Vaut mieux le faire maintenant si possible, on a moins de chance d'en oublier comme ça

**Review by qperie — APPROVED** [review]

**Review by JinDouang — APPROVED** [review]

**Pull request merged**